azkaban-web-server/src/main/java/azkaban/scheduler/QuartzScheduler.java

```diff
@@ -55,7 +55,7 @@ public QuartzScheduler(final Props azProps) throws SchedulerException {
       return;
     }
     final StdSchedulerFactory schedulerFactory =
-        new StdSchedulerFactory(azProps.toProperties());
+        new StdSchedulerFactory(azProps.toAllProperties());
```

Comment:
what do you think of reading quartz only properties with azProps.get() vs dumping every properties?
also i don't think we should specify mysql username/password twice in private property file. How about reading mysql username/password from azProps and convert them into quartz properties?

Reply:
Synced up offline. I agree. We may convert quartz properties in Guice Module code.
Adding a todo here for reminder.